content/_code-samples/checks/py/cash_check.py

```diff
@@ -0,0 +1,80 @@
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.utils import xrp_to_drops
+from xrpl.wallet import Wallet
+
+from xrpl.clients import JsonRpcClient
+from xrpl.models import CheckCreate, IssuedCurrencyAmount
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234")
+
+
+# cash an xrp check
+
+# check casher seed
+sender_seed = "sxxxxxxxxxxxxxxxxxxxxx"
+
+# check id
+check_id = str
+
+# amount to cash
+amount = 10.00
+
+# generate wallet from seed
+sender_wallet = Wallet(seed=sender_seed, sequence=0)
```

Comment:
Please generate wallets using the generate_faucet_wallet function instead of hardcoding seeds. 
https://xrpl-py.readthedocs.io/en/v1.0.0/source/xrpl.wallet.html?highlight=faucet#xrpl.wallet.generate_faucet_wallet